odkx-src/sync-endpoint.rst

```diff
@@ -132,3 +143,153 @@ Assigning users to groups
 
     a. Navigate to the :guilabel:`memberUid` section.
     b. Click modify group members to manage members.
+
+.. _sync-endpoint-advanced:
+
+Advanced
+--------
+
+.. _sync-endpoint-ldap-defaults:
+
+Editing the defaults of LDAP Directory
+"""""""""""""""""""""""""""""""""""""""""""""
+    Modify the :file:`ldap.env` file to configure the environment variables. The :file:`ldap.env` file is located in the sync-endpoint-default-setup directory.
+
+    The default settings are as follows 
+
+     .. code-block:: console
+
+      # openldap
+      LDAP_ORGANISATION=Open Data Kit            // name of your organisation
+      LDAP_DOMAIN=example.org                    // domain of your organisation
+      LDAP_READONLY_USER=true                    // enable the read only user
+      LDAP_READONLY_USER_PASSWORD=readonly       // password for read only user
+      LDAP_ADMIN_PASSWORD=admin                  // default password for admin account
+
+      # phpldapadmin
+      PHPLDAPADMIN_LDAP_HOSTS=ldap-service   // This is for the phpLDAPadmin . In Docker Swarm this is the hostname of the service running LDAP . This can be 
+                                                eddited in the docker-compose.yml file
+
+
+  .. note::
+
+    For LDAP environment variables the corresponding options in the security.properties also need to be modified . The security.properties file is
+    located at config/sync-endpoint in the sync-endpoint-default-setup directory.
+
+
+.. _sync-endpoint-custom-ldap:
+
+Using a Different LDAP Directory
```

Comment:
I just realized a similar section already exists at https://docs.odk-x.org/sync-endpoint-manual-setup/#custom-ldap-directory. Could you try to merge your content with the existing section?